HPXMLtoOpenStudio/resources/hvac_sizing.rb

```diff
@@ -1696,6 +1715,44 @@ def self.apply_hvac_duct_loads_cooling(mj, zone, hvac_loads, zone_loads, all_spa
     end
   end
 
+  def self.apply_hvac_cfis_loads(mj, hvac_loads, zone_loads, hvac_heating, hvac_cooling, hpxml_bldg)
+    '''
+    CFIS Ventilation Loads
+    '''
+    if (not hvac_heating.nil?) && (not hvac_heating.distribution_system.nil?)
+      hvac_distribution = hvac_heating.distribution_system
+    elsif (not hvac_cooling.nil?) && (not hvac_cooling.distribution_system.nil?)
+      hvac_distribution = hvac_cooling.distribution_system
+    end
+    return if hvac_distribution.nil?
+
+    vent_mech_cfis = hpxml_bldg.ventilation_fans.find { |vent_mech| vent_mech.fan_type == HPXML::MechVentTypeCFIS && vent_mech.distribution_system_idref == hvac_distribution.id }
+    return if vent_mech_cfis.nil?
+
+    vent_cfm = vent_mech_cfis.total_unit_flow_rate
```

Comment:
Also, should we exclude the cfis cfms from the ventilation load calculation (q_bal_cfm)? I feel we do want it for infiltration adjustments considering it's analyzing all the air flows in the house.